test/models/Reads/AddressReadsSpec.scala

```diff
@@ -115,6 +115,13 @@ class AddressReadsSpec extends WordSpec with MustMatchers with OptionValues with
 
           result.asInstanceOf[UkAddress].postalCode mustBe ukAddressSample.postalCode
         }
+
+        "with isChanged flag" in {
+          val modifiedAddress = address + ("isChanged" -> JsBoolean(true))
+          val result = modifiedAddress.as[Address]
+
+          result.asInstanceOf[UkAddress].isChanged.value mustBe true
```

Comment:
Nicer to say ... isChanged mustBe Some(true) - won't get an exception if fails then just a mismatch between values. Also, I'd add a test for result.asInstanceOf[UkAddress].isChanged mustBe None where there is no isChanged flag.